doctor/lib/ocr_utils.py

```diff
@@ -0,0 +1,345 @@
+import re
+
+import pdfplumber
+from pdfplumber.ctm import CTM
+import pytesseract
+from pytesseract import Output
+import pandas as pd
+from PIL import Image
+
+
+def deskew(obj: dict) -> bool:
+    """Remove skewed text from a page
+
+    CTM stands for current transformation matrix.
+    Pdf plumber has a method to calculate the angle of text which we use here
+
+    Traditionally this is only seen in circular stamps which confuses the
+    content, or in perpendicular text of the ninth circuit courts which also
+    confuses the text.
+    """
+    if (matrix := obj.get("matrix")) is None:
+        return True
+
+    # Remove Skew
+    my_char_ctm = CTM(*matrix)
+    if my_char_ctm.skew_x != 0:
+        return False
+    return True
+
+
+def get_page_text(page: pdfplumber.PDF.pages, strip_margin: bool) -> str:
+    """Extract page text
+
+    Using pdf plumber extract out the text of the document that is not
+    skewed (ie a stamp of approval) and extract out text removing blue text
+
+    :param page: pdf plumber page
+    :param strip_margin: a flag to crop out the margin of a document and skewed content
+    :return: text
+    """
+    if strip_margin:
+        # Crop margins and remove skewed text
+        _, _, width, height = page.bbox
+        bbox = (
```

Comment:
Hm, I think you're computing the pixels_per_inch, right, then doing the math? 

What if you computed the px/inch first, then...

```
pixels_per_inch = width / 8.5
bbox = (
    pixels_per_inch * 1,  # One inch from left edge
    pixels_per_inch * 1,  # One inch down from top
    pixels_per_inch * 7.5,  # 7.5 inches from left edge (one inch from right)
    pixels_per_inch * 10,  # 10 inches from top (one inch from bottom)
)
```